### DISCUSSION TITLE
Using script with YouTube ReVanced

### DISCUSSION THREAD

**Discussion by masterwishx** [ilyhalight/voice-over-translation#281] [Q&A] [answered]
How to use script with YouTube ReVanced?

**Comment by NullVerdict** [comment]
> How to use script with YouTube ReVanced?

Well, not yet. At least I don't know how to run userscripts. You can ask them.

**Comment by masterwishx** [comment]
OK , Thanks Working fine on Chrome, imported to AdGuard the Cloudflare script instead of Tampermonkey.

**Comment by ilyhalight** [comment, marked as answer]
Unfortunately, Revanced does not support user scripts. However, one of the users of the extension (#187) suggested to the developers of Revanced to add the possibility of translation, but so far there has been no response

**Reply by masterwishx** [reply]
Thanks, got it
